backend/protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -146,7 +146,6 @@ enum RunnerState {
 }
 
 message RegisterRunnerRequest {
-  // UUID representing the runner instance.
```

Comment:
I think this is supposed to update the identifical comment in ftl_pb.ts and ftl.pb.go? Running `bit` didn't seem to trigger anything